packages/ui/src/components/molecules/Dropdown/Dropdown.tsx

```diff
@@ -135,7 +134,12 @@ export const Dropdown: React.FC<DropdownProps> = ({
           }
           onClick={toggleDropdown}
           $restrictedItem
+          leftImage={selectedDropdownItem.leftImage}
+          rightText={selectedDropdownItem.rightText}
```

Comment:
No need for these, as we are already using spread operator above.

```
          {...selectedDropdownItem}
```

Reply:
done